soc/infineon/cat1b/cyw20829/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Infineon CYW20829 based MCU default configuration
+
+if SOC_DIE_CYW20829
+
+config NUM_IRQS
+	default 32
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 48000000
```

Comment:
could you check that SYS_CLOCK_HW_CYCLES_PER_SEC= 48000000 and  NUM_IRQS = 32 are ok here?

Reply:
fixed the NUM_IRQ